git_machete/git_operations.py

```diff
@@ -179,17 +179,17 @@ def __init__(self) -> None:
         self.__contains_equivalent_tree_cached: Dict[Tuple[FullCommitHash, FullCommitHash], bool] = {}
 
     @staticmethod
-    def _run_git(git_cmd: str, *args: str, **kwargs: Dict[str, str]) -> int:
+    def _run_git(git_cmd: str, *args: str, **kwargs: Any) -> int:
```

Comment:
Actually... why hasn't `mypy` capture that problem so far, unlike in the [blog post you've linked](https://adamj.eu/tech/2021/05/11/python-type-hints-args-and-kwargs/)? 🤔 is it possible to reconfigure our mypy to capture this type of errros?

Reply:
Both `pycharm` and `mypy` captured the problem while trying to do https://github.com/VirtusLab/git-machete/pull/383, so there is no need to configure anything. 
![image](https://user-images.githubusercontent.com/86779115/151423915-42deb9e3-4e62-4afd-a1d3-8598cea1fe8c.png)

Reply:
Oh! okay, as in, `mypy` captures the problem but at **call site** (which has only been added now right?) and not at the **declaration**

Reply:
That's fine, let's leave as is